src/nvim/ex_cmds.c

```diff
@@ -67,6 +81,10 @@ typedef struct sign sign_T;
 
 #ifdef INCLUDE_GENERATED_DECLARATIONS
 # include "ex_cmds.c.generated.h"
+#include "ex_cmds_defs.h"
+#include "regexp_defs.h"
+#include "pos.h"
+
```

Comment:
Includes should be above this `#ifdef` block.  Also missing 'nvim/' prefix in file path.